component/background/background.tsx

```diff
@@ -58,16 +56,4 @@ class StorageService {
 }
 
 (window as any).BackgroundStorageService = new StorageService();
-
-// Initialize Firebase
-const config = {
-  apiKey: "AIzaSyBakHeV8lMlysuBRtIWU9vz_hv6dF_zHxM",
-  authDomain: "url-connet.firebaseapp.com",
-  databaseURL: "https://url-connet.firebaseio.com",
-  messagingSenderId: "1089725560944",
-  projectId: "url-connet",
-  storageBucket: "url-connet.appspot.com"
-};
-firebase.initializeApp(config);
-
-(window as any).firebaseDB = firebase.database();
+(window as any).room = new RoomService(id);
```

Comment:
The room is usually a class, while room means an instance.
I choose room because it is a singleton object rather than a class